manifests/s/SteelSeries/GG/31.0.0/SteelSeries.GG.installer.yaml

```
@@ -0,0 +1,20 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=QUSU.7-2-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: SteelSeries.GG
+PackageVersion: 31.0.0
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+Scope: machine
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://engine.steelseriescdn.com/SteelSeriesGG31.0.0Setup.exe
+  InstallerSha256: 36f245ac9c96c90bd056a1dc0222b244e5b17c0f878234bbc5f0a18dc6ed5bc3
```

Hello @al-lac, the correct hash for this .exe is `e5f74d1a6cefd3242f679482a4fbaa81058d23662a0762906d2b2b952293178a`.

```suggestion
  InstallerSha256: e5f74d1a6cefd3242f679482a4fbaa81058d23662a0762906d2b2b952293178a
```

---

You are totally right, sorry about that!

---

No worries! 🙂